components/Cards/Card.tsx

```diff
@@ -19,7 +24,13 @@ const Card: FC<{ data: IData }> = (props) => {
           </h2>
           <CopyToClipboard url={url} />
         </header>
-        <p className="h-24 w-full overflow-hidden font-sans">{description}</p>
+        <div className="h-[7rem]">
```

Comment:
```suggestion
        <div className="h-[5rem]">
```

Reply:
@rupali-codes probably 5rem height can produce some ui misalignment, I am sure that 7rem will work well